docs/UserGuide.md

```diff
@@ -460,6 +460,12 @@ Example:
 ![RedoCommandExample](images/RedoCommandExample.png)
 
 <br>
+<div markdown="span" class="alert alert-warning">
+**:exclamation: Caution:**
+Once the user exits the app, all temporarily saved states of the tracker will be deleted!
```

Comment:
A little nitpicking here. I think it might be better to rephrase it as the following:

`Only the current state will be stored upon exiting the app.`
`The temporary storage that stores the undo as well as redo states are not stored anywhere upon exit.`